ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-release-4.7.yaml

```diff
@@ -3,33 +3,21 @@ base_images:
     name: "4.7"
     namespace: ocp
     tag: base
-  ocp_builder_rhel-7-golang-1.13:
+  ocp_builder_rhel-8-golang-1.14-openshift-4.7:
     name: builder
     namespace: ocp
-    tag: rhel-7-golang-1.13
-  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.15-openshift-4.6
-  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.15-openshift-4.7
-  ocp_builder_rhel-8-golang-openshift-4.6:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-openshift-4.6
+    tag: rhel-8-golang-1.14-openshift-4.7
 binary_build_commands: make build --warn-undefined-variables
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.14
 images:
 - dockerfile_path: Dockerfile.rhel
   from: base
   inputs:
-    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    ocp_builder_rhel-8-golang-1.14-openshift-4.7:
```

Comment:
why are you downgrading this?

Reply:
also you the fact that the version here is misaligned w/ the version in the `as` seems like a bad idea, since it implies that you're expecting to substitute a reference to "golang-1.15" in the dockerfile, with something you've called "golang-1.14".